src/components/TimeCardPage/TimeSheet.js

```diff
@@ -49,33 +47,91 @@ export default function Page() {
         // Callback for date-range picker - does any pre-processing when grabbing a new date 
         setStartDate(startDate); 
         setEndDate(endDate); 
```

Comment:
❓ What does this do? It looks like it's just re-setting the same value?

Reply:
I think thats a mistake, i changed it to set start and end based on values passed in.

Reply:
and we don't have a solidified plan on how we plan to serve timesheets